bs-react-native-next/src/apis/ToastAndroid.md

```diff
@@ -0,0 +1,38 @@
+---
+id: apis/ToastAndroid
+title: ToastAndroid
+wip: true
+---
+
+```reason
+type message = string;
+type duration;
+type gravity;
+
+[@bs.module "react-native"] [@bs.scope "ToastAndroid"] [@bs.val]
```

Comment:
Nice work! 😀

Here, too, I think the [@bs.val] are superfluous though.

Reply:
Thanks, they were.